src/Analysis/GWASV2/GWASContainer.jsx

```diff
@@ -1,100 +1,87 @@
-import "./GWASV2.css";
+import React, { useState } from 'react';
+import { Space, Button, Popconfirm } from 'antd';
+import SelectStudyPopulation from './SelectStudyPopulation/SelectStudyPopulation';
+import ProgressBar from './Shared/ProgressBar/ProgressBar';
+import { gwasV2Steps } from './Shared/constants';
+import './GWASV2.css';
 
 const GWASContainer = () => {
   const [current, setCurrent] = useState(0);
   const [
     selectedStudyPopulationCohort,
     setSelectedStudyPopulationCohort,
   ] = useState({});
-  const gwasSteps = [
-    {
-      title: "Step 1",
-      description: "Select Study Population",
-    },
-    {
-      title: "Step 2",
-      description: "Select Outcome Phenotypes",
-    },
-    {
-      title: "Step 3",
-      description: "Select Covariate Phenotype",
-    },
-    {
-      title: "Step 4",
-      description: "Configure GWAS",
-    },
-  ];
 
   const generateStep = () => {
     // steps 2 & 3 very similar
     switch (current) {
-      case 0:
-        // select study population
-        return (
-          <SelectStudyPopulation
-            selectedStudyPopulationCohort={selectedStudyPopulationCohort}
-            setSelectedStudyPopulationCohort={setSelectedStudyPopulationCohort}
-            current={current}
-          />
-        );
-      case 1:
-        // outcome (customdichotomous or not)
-        return <React.Fragment>step 2</React.Fragment>;
-      case 2:
-        // covariates (customdichtomous or not)
-        return <React.Fragment>step 3</React.Fragment>;
-      case 3:
-        // all other input (mafs, imputation, etc), review, and submit
-        return <React.Fragment>step 4</React.Fragment>;
-      default:
-        // required for eslint
-        return null;
+    case 0:
+      // select study population
+      return (
+        <SelectStudyPopulation
+          selectedStudyPopulationCohort={selectedStudyPopulationCohort}
+          setSelectedStudyPopulationCohort={setSelectedStudyPopulationCohort}
+          current={current}
+        />
+      );
+    case 1:
+      // outcome (customdichotomous or not)
+      return <React.Fragment>step 2</React.Fragment>;
+    case 2:
+      // covariates (customdichtomous or not)
+      return <React.Fragment>step 3</React.Fragment>;
+    case 3:
+      // all other input (mafs, imputation, etc), review, and submit
+      return <React.Fragment>step 4</React.Fragment>;
+    default:
+      // required for eslint
+      return null;
     }
   };
   return (
     <React.Fragment>
+      <ProgressBar current={current} />
       {/* Inline style block needed so centering rule doesn't impact other workflows */}
-      <style>{`.analysis-app__actions > div:nth-child(1) {margin: 0 auto; }`}</style>
-      <div className="GWASV2">
-        <Space direction={"vertical"} style={{ width: "100%" }}>
-          <div className="steps-content">
+      <style>
+        {'.analysis-app__actions > div:nth-child(1) { width: 100%; }'}
+      </style>
```

Comment:
can you include this styling without it cascading to other steps

Reply:
Per our discussion, let's hold off on this. It results in the progress bar not taking up the full width. My guess is because the code changing the margins have been removed, it won't impact the subsequent steps

<img width="1470" alt="image" src="https://user-images.githubusercontent.com/113449836/196984259-9a433760-e6a5-4024-8b97-bd4238ea48bd.png">